Gemfile.base

```diff
@@ -74,12 +74,8 @@ gem 'htmlentities', '~>4.3', '>= 4.3.4'
 gem 'inherited_resources', '~> 1.7.2'
 gem 'json', '~> 1.8.1'
 
-case ENV.fetch('DATABASE', 'mysql')
-when 'postgresql'
-  gem 'pg'
-when 'mysql'
-  gem 'mysql2', '~>0.3.11'
-end
+gem 'mysql2', '~>0.3.11'
+gem 'pg', '~> 0.21.0'
```

Comment:
Why not using `install_if`to actually not install it when not needed?

Reply:
So we don't need to keep multiple docker images.